### ISSUE TITLE
Can't Analyze groovy file

### ISSUE THREAD

**Issue by b1zzu** [nvuillam/vscode-groovy-lint#28]
groovy lint file doesn't report any lint errors and the console is reporting this error:
```
2020-04-27T19:34:54.853Z vscode-groovy-lint Save event received for file:///home/b1zzu/Projects/gitlab.cee.redhat.com/integreatly-qe/ci-cd/pipelines/rhmi-release.groovy
2020-04-27T19:35:08.510Z vscode-groovy-lint Request execute command {"command":"groovyLint.lint","arguments":[]}
2020-04-27T19:35:08.510Z vscode-groovy-lint Request execute npm-groovy-lint for file:///home/b1zzu/Projects/gitlab.cee.redhat.com/integreatly-qe/ci-cd/pipelines/rhmi-release.groovy with options {"force":true}
2020-04-27T19:35:08.510Z vscode-groovy-lint Reset diagnostics for file:///home/b1zzu/Projects/gitlab.cee.redhat.com/integreatly-qe/ci-cd/pipelines/rhmi-release.groovy
2020-04-27T19:35:08.510Z npm-groovy-lint <<< NpmGroovyLint.run START >>>
Start analyzing file:///home/b1zzu/Projects/gitlab.cee.redhat.com/integreatly-qe/ci-cd/pipelines/rhmi-release.groovy
2020-04-27T19:35:08.512Z npm-groovy-lint GroovyLint used config file: /home/b1zzu/Projects/gitlab.cee.redhat.com/integreatly-qe/ci-cd/.groovylintrc.json
VsCode Groovy Lint error: The "path" argument must be of type string. Received type object
Error: The "path" argument must be of type string. Received type object
    at NpmGroovyLint.preProcess (/home/b1zzu/.vscode/extensions/nicolasvuillamy.vscode-groovy-lint-0.9.3/server/node_modules/npm-groovy-lint/jdeploy-bundle/groovy-lint.js:136:23)
2020-04-27T19:35:08.514Z vscode-groovy-lint Error processing file:///home/b1zzu/Projects/gitlab.cee.redhat.com/integreatly-qe/ci-cd/pipelines/rhmi-release.groovyThe "path" argument must be of type string. Received type object
Error: The "path" argument must be of type string. Received type object
    at NpmGroovyLint.preProcess (/home/b1zzu/.vscode/extensions/nicolasvuillamy.vscode-groovy-lint-0.9.3/server/node_modules/npm-groovy-lint/jdeploy-bundle/groovy-lint.js:136:23)
```
I tried to lint the same file using `npm-groovy-lint` and it works

Do you guys have any idea why this happened?

**Comment by nvuillam** [comment]
VsCode Groovy Lint is one version away of npm-groovy-lint
Is you repo sensitive, or could you share it (privately if u prefer) so I
can reproduce the issue ? :)

Le lun. 27 avr. 2020 à 21:38, Davide Bizzarri <notifications@github.com> a
écrit :
